gradle/buildsrc.libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
-asciidoctor-gradle-jvm = '4.0.2'
-assetpipeline = '5.0.1'
+asciidoctor-gradle-jvm = '4.0.3'
+assetpipeline = '5.0.4'
```

Comment:
assetpipeline is in the bom, right?  Do we need it here?  5.0.5 is the version we should use to avoid that JS error

Reply:
Actually `asset-pipeline` is only used in a test app, and we don't need the gradle plugin there. So, it can be removed here and in the build dependencies of that test app.

Reply:
We can merge this then.  And create a ticket to do this later?

Reply:
Good point.  Removed it entirely.